resources/deb/postinst.erb

```diff
@@ -2,7 +2,7 @@
 <%- get_services.each do |service| -%>
   # switch based on systemd vs systemv
   #
-  <%- if service.init_system.nil? || service.init_system.eq?('systemd') -%>
+  <%- if service.init_system.nil? || service.init_system.eql?('systemd') -%>
```

Comment:
non-blocking, but curious about the functional difference between `eq` and `eql` here

Reply:
had this error:
```
Generated: /tmp/d20210202-87398-wo6b1m/debian/preinst
undefined method `eq?' for "systemd":String
Did you mean?  eql?
(erb):5:in `block in generate_packaging_artifacts'
(erb):2:in `each'
(erb):2:in `generate_packaging_artifacts'
/usr/local/rvm/rubies/ruby-2.6.4/lib/ruby/2.6.0/erb.rb:901:in `eval'
/usr/local/rvm/rubies/ruby-2.6.4/lib/ruby/2.6.0/erb.rb:901:in `result'
/home/cbadescu/puppetlabs/vanagon/lib/vanagon/utilities.rb:294:in `erb_string'
/home/cbadescu/puppetlabs/vanagon/lib/vanagon/utilities.rb:307:in `erb_file'
/home/cbadescu/puppetlabs/vanagon/lib/vanagon/platform/deb.rb:45:in `block in generate_packaging_artifacts'
/home/cbadescu/puppetlabs/vanagon/lib/vanagon/platform/deb.rb:44:in `each'
/home/cbadescu/puppetlabs/vanagon/lib/vanagon/platform/deb.rb:44:in `generate_packaging_artifacts'
/home/cbadescu/puppetlabs/vanagon/lib/vanagon/project.rb:683:in `generate_packaging_artifacts'
/home/cbadescu/puppetlabs/vanagon/lib/vanagon/driver.rb:147:in `run'
/home/cbadescu/puppetlabs/vanagon/lib/vanagon/cli/build.rb:51:in `block in run'
/home/cbadescu/puppetlabs/vanagon/lib/vanagon/cli/build.rb:48:in `each'
/home/cbadescu/puppetlabs/vanagon/lib/vanagon/cli/build.rb:48:in `run'
/home/cbadescu/puppetlabs/vanagon/lib/vanagon/cli.rb:82:in `run'
/home/cbadescu/puppetlabs/vanagon/bin/vanagon:7:in `<top (required)>'
/home/cbadescu/.rvm/gems/ruby-2.6.4/bin/vanagon:23:in `load'
/home/cbadescu/.rvm/gems/ruby-2.6.4/bin/vanagon:23:in `<main>'
Traceback (most recent call last):
	18: from /home/cbadescu/.rvm/gems/ruby-2.6.4/bin/vanagon:23:in `<main>'
	17: from /home/cbadescu/.rvm/gems/ruby-2.6.4/bin/vanagon:23:in `load'
	16: from /home/cbadescu/puppetlabs/vanagon/bin/vanagon:7:in `<top (required)>'
	15: from /home/cbadescu/puppetlabs/vanagon/lib/vanagon/cli.rb:82:in `run'
	14: from /home/cbadescu/puppetlabs/vanagon/lib/vanagon/cli/build.rb:48:in `run'
	13: from /home/cbadescu/puppetlabs/vanagon/lib/vanagon/cli/build.rb:48:in `each'
	12: from /home/cbadescu/puppetlabs/vanagon/lib/vanagon/cli/build.rb:51:in `block in run'
	11: from /home/cbadescu/puppetlabs/vanagon/lib/vanagon/driver.rb:147:in `run'
	10: from /home/cbadescu/puppetlabs/vanagon/lib/vanagon/project.rb:683:in `generate_packaging_artifacts'
	 9: from /home/cbadescu/puppetlabs/vanagon/lib/vanagon/platform/deb.rb:44:in `generate_packaging_artifacts'
	 8: from /home/cbadescu/puppetlabs/vanagon/lib/vanagon/platform/deb.rb:44:in `each'
	 7: from /home/cbadescu/puppetlabs/vanagon/lib/vanagon/platform/deb.rb:45:in `block in generate_packaging_artifacts'
	 6: from /home/cbadescu/puppetlabs/vanagon/lib/vanagon/utilities.rb:307:in `erb_file'
	 5: from /home/cbadescu/puppetlabs/vanagon/lib/vanagon/utilities.rb:294:in `erb_string'
	 4: from /usr/local/rvm/rubies/ruby-2.6.4/lib/ruby/2.6.0/erb.rb:901:in `result'
	 3: from /usr/local/rvm/rubies/ruby-2.6.4/lib/ruby/2.6.0/erb.rb:901:in `eval'
	 2: from (erb):2:in `generate_packaging_artifacts'
	 1: from (erb):2:in `each'
(erb):5:in `block in generate_packaging_artifacts': undefined method `eq?' for "systemd":String (NoMethodError)
Did you mean?  eql?

```